src/gc/null.cr

```diff
@@ -13,6 +13,21 @@ fun __crystal_realloc(ptr : Void*, size : UInt32) : Void*
   LibC.realloc(ptr, size)
 end
 
+# :nodoc:
+fun __crystal_malloc64(size : UInt64) : Void*
+  LibC.malloc(size)
+end
+
+# :nodoc:
+fun __crystal_malloc_atomic64(size : UInt64) : Void*
+  LibC.malloc(size)
```

Comment:
here should be LibGC.malloc_atomic?

Reply:
This is the null gc, which doesn't do any gc, it just forwards to the libc malloc.